backend/src/main/java/se/chalmers/dat265/group1/storage/FileUtil.java

```diff
@@ -0,0 +1,12 @@
+package se.chalmers.dat265.group1.storage;
+
+import java.io.*;
+
+public class FileUtil {
+    public static void write(byte[] bytes, String filePath) throws IOException {
+        OutputStream os = new FileOutputStream(filePath);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "FileOutputStream" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2095)